test/integration/helpers/shared_examples.rb

```diff
@@ -0,0 +1,32 @@
+if os[:release].to_i < 7
```

Comment:
While this is OK, I'd rather we use the InSpec described method using the [os resource](https://www.inspec.io/docs/reference/resources/os/) and change it to the following:
```suggestion
if os.release.to_i < 7
```

Reply:
I've committed the above change suggestion, but I couldn't find where else I need to change based on your comment of "use the InSpec described method using the os resource". Could you elaborate a little bit more?